torch/csrc/autograd/functions/batch_normalization.cpp

```diff
@@ -84,6 +84,11 @@ auto BatchNormForward::apply(const variable_list& inputs) -> variable_list {
 
 auto BatchNormBackward::apply(const variable_list& grad_outputs) -> variable_list {
   auto& input = this->input.unpack();
+
+  if (!input) throw std::runtime_error("Trying to backward through the "
```

Comment:
i'm not too happy about the copy-pasta string. I want to make it a #define. What's a good place to put it?